exporters/jaeger/src/recordable.cc

```diff
@@ -0,0 +1,183 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <opentelemetry/exporters/jaeger/recordable.h>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace jaeger
+{
+
+Recordable::Recordable() : span_{new thrift::Span} {}
+
+void Recordable::PopulateAttribute(nostd::string_view key,
+                                   const opentelemetry::common::AttributeValue &value)
+{
+  if (nostd::holds_alternative<int64_t>(value))
+  {
+    AddTag(std::string{key}, nostd::get<int64_t>(value));
+  }
+  else if (nostd::holds_alternative<bool>(value))
+  {
+    AddTag(std::string{key}, nostd::get<bool>(value));
+  }
+  else if (nostd::holds_alternative<double>(value))
+  {
+    AddTag(std::string{key}, nostd::get<double>(value));
+  }
+  else if (nostd::holds_alternative<nostd::string_view>(value))
+  {
+    AddTag(std::string{key}, std::string{nostd::get<nostd::string_view>(value)});
+  }
+  // TODO: extend other AttributeType to the types supported by Jaeger.
+}
+
+void Recordable::SetIds(trace::TraceId trace_id,
```

Comment:
This method is changed to SetIdentity(). Probably there would be merge conflict.